microsetta_private_api/db/patches/0139.sql

```diff
@@ -0,0 +1,28 @@
+-- May 13, 2024
+-- Create two tables for observation categories and observation errors list
+-- Add observations column to barcode_scans table
+CREATE TABLE observation_categories (
+    categories VARCHAR(255) PRIMARY KEY
```

Comment:
```suggestion
    category VARCHAR(255) PRIMARY KEY
```

Prefer to keep column names in the same singular/plural form as the data they're storing.